csc302-spring-2021/proj-DaTeam
# docs(openapi): add parser examples and partial schema

## Description



- Fixes OpenAPI doc
- Add partial schema for XML body
- Add two examples from Cancer Care Ontario

## Checklist


## QA Steps



1. `npm run clean:ci`
2. `npm install`
3. `npm run build`
1. `cd backend && npm run dev`
1. Go to http://localhost:3001/openapi
1. Verify that `POST /parser` shows two examples
